library/build.gradle

```diff
@@ -1,6 +1,9 @@
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
 apply plugin: 'signing'
+apply plugin: 'com.github.dcendents.android-maven'
+
+group='com.github.WhistleLabs'
```

Comment:
Currently in bintray:
`com.whistle.custom:pusher-java-client:1.0.0`
`com.whistle.custom:cropimage:1.0.1`
`com.whistle.twine:twine-core:0.0.6`
`com.whistle.whistlecoredroid:whistlecoredroid:3.0.11`

(is totally ok that its jitpack for now)